keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,73 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes):
+    """Pads the given bounding boxes to convert it from a ragged tensor to
+    a regular tensor.
```

Comment:
Pads the given bounding box tensor with -1s.  This is done to convert RaggedTensors to be converted into standard Dense tensors, which have better performance and compatibility within the TensorFlow ecosystem.

Reply:
updated!